Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/ImageClassifier/ImageClassifier.xaml.cs

```diff
@@ -0,0 +1,448 @@
+                                                                                                new float[] { 0.229f, 0.224f, 0.225f}) },
+                    { Classifier.Inception_V1,      null }, // No preprocessing required
+                    { Classifier.Inception_V2,      null }, // ????
+                    { Classifier.ShuffleNet_V1,     () => TensorizationModels.Normalize0_1ThenZScore(224, 224, 4,
+                                                                                                new float[] { 0.485f, 0.456f, 0.406f },
+                                                                                                new float[] { 0.229f, 0.224f, 0.225f}) },
+                    { Classifier.ShuffleNet_V2,     () => TensorizationModels.Normalize0_1ThenZScore(224, 224, 4,
+                                                                                                new float[] { 0.485f, 0.456f, 0.406f },
+                                                                                                new float[] { 0.229f, 0.224f, 0.225f}) },
+                    { Classifier.EfficientNetLite4, () => TensorizationModels.NormalizeMinusOneToOneThenTransposeNHWC() },
+                    // Large Models
+                    { Classifier.AlexNet,           null }, // No preprocessing required
+                    { Classifier.CaffeNet,          null }, // No preprocessing required
+                    { Classifier.RCNN_ILSVRC13,     null }, // No preprocessing required
+                    { Classifier.ResNet,            () => TensorizationModels.Normalize0_1ThenZScore(224, 224, 4,
+                                                                                                new float[] { 0.485f, 0.456f, 0.406f },
+                                                                                                new float[] { 0.229f, 0.224f, 0.225f}) },
+                    { Classifier.VGG,               () => TensorizationModels.Normalize0_1ThenZScore(224, 224, 4,
+                                                                                                new float[] { 0.485f, 0.456f, 0.406f },
+                                                                                                new float[] { 0.229f, 0.224f, 0.225f}) },
+                    { Classifier.ZFNet512,          null }, // No preprocessing required
+                };
+            }
+
+            InitializeWindowsMachineLearning();
+        }
+
+        private void InitializeWindowsMachineLearning()
+        {
+            _tensorizationSession =
+                CreateLearningModelSession(
+                    TensorizationModels.BasicTensorization(
+                        224, 224,
+                        1, 4, CurrentImageDecoder.PixelHeight, CurrentImageDecoder.PixelWidth,
+                        "nearest"),
+                        LearningModelDeviceKind.Cpu);

```

Comment:
shouldn't this come from the combo box value?

Reply:
It's possible that @smk2007 always wanted tensorization to be done on CPU?

Reply:
It is slower on dml.